src/progress_bars/ProgressBars.test.tsx

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import {ProgressBar, ProgressBarProps, ProgressBarType, ProgressBarStatus, ProgressBarAnimation} from ".";
+import {shallow, mount, ShallowWrapper} from "enzyme";
+import * as React from "react";
+
+let wrapper: ShallowWrapper<ProgressBarProps>;
+
+describe("<ProgressBar /> rendering", () => {
+    beforeAll(() => {
+        wrapper = shallow<ProgressBar>(<ProgressBar value={10} max={100} style={{width: "50%"}} />);
+    });
+
+    it("renders correctly", () => {
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("<ProgressBar /> rendering with label", () => {
+    beforeAll(() => {
+        wrapper = shallow<ProgressBar>(<ProgressBar value={10} max={100} labeled={true} style={{width: "50%"}} />);
+    });
+
+    it("renders correctly", () => {
```

Comment:
Please provide proper message here for all test cases.

Reply:
Addressed the comment